pkgs/development/php-packages/datadog_trace/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPecl, curl, fetchurl, lib, pcre2 }:
+
+buildPecl rec {
+  pname = "ddtrace";
+  version = "0.70.0";
+
+  src = fetchurl {
+    url = "http://pecl.php.net/get/datadog_trace-${version}.tgz";
```

Comment:
Changed!

Reply:
@drupol But this isn't true. Just look at #162442. The bot uses https://repology.org/ to find updates. I think someone said in some previous thread that we do prefer the "upstream's preferred release channel", which I would think is the pecl.

So if anything, we could have ditched the `src` attribute all together and just had `version` and `sha256`.

Reply:
The reason it's not just version and sha is because the url pattern for this extension doesn't conform to convention. 

The pname is "ddtrace" but url to the package is "datadog_trace". The default behavior uses the pname in the url. 

What would you like me to do? I see examples of both methods in other extensions.